desktop/onionshare/connection_tab.py

```diff
@@ -326,11 +326,20 @@ def tor_con_fail(self, msg):
         """
         self.tor_con.hide()
 
-        # If we're on first launch, switch to use bridge
+        # If there is a message, update the text of the bridge widget
+        if msg:
+            self.use_bridge_widget.connection_error_message.setText(f"{msg}")
+
+        # If we're on first launch, check if wasCancelled
+        # If cancelled, stay in first launch widget and show buttons
```

Comment:
Tiny nitpick but the method is named `wasCanceled()` (not `wasCancelled()`), I guess we should be consistent with US spelling even though I prefer the two l's :)